package.json

```diff
@@ -3,14 +3,16 @@
   "version": "0.8.0",
   "description": "Build a docker composition",
   "main": "build/index.js",
-  "types": "lib/index.ts",
+  "types": "build/index.d.ts",
   "scripts": {
+    "clean": "rm -rf build",
```

Comment:
We usually use `rimraf` which is cross-platform.

Reply:
Ah, that's nice. I'll add that in in a upcoming PR - thanks!